cmd/create/cluster/cmd.go

```diff
@@ -655,6 +655,10 @@ func getVersionList(client *cmv1.Client, channelGroup string) (versionList []str
 		versionList = append(versionList, strings.Replace(v.ID(), "openshift-v", "", 1))
 	}
 
+	if len(versionList) == 0 {
+		err = fmt.Errorf("Failed to find versions for the provided channel-group: '%s'.", channelGroup)
```

Comment:
/nitpick
I would add a `return` inside this block just to be consistent. If someone else decides to add more code after this block, they might cause an unintended bug

Reply:
Done. thx!